geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/TestWithDataStore.scala

```diff
@@ -32,72 +26,55 @@ import scala.collection.JavaConverters._
  */
 trait TestWithDataStore extends Specification {
 
-  def spec: String
-  def dtgField: Option[String] = Some("dtg")
-
-  def additionalDsParams(): Map[String, Any] = Map.empty
-
   // we use class name to prevent spillage between unit tests in the mock connector
```

Comment:
Update comment!  (Just showing that I'm reading:).)

May be worth doing a 'git grep -i mock' before we call things good.

Reply:
went through `git grep` and removed stuff